examples/seismic/acoustic/operators.py

```diff
@@ -124,13 +150,21 @@ def AdjointOperator(model, geometry, space_order=4,
 def GradientOperator(model, geometry, space_order=4, save=True,
                      kernel='OT2', **kwargs):
     """
-    Constructor method for the gradient operator in an acoustic media
-
-    :param model: :class:`Model` object containing the physical parameters
-    :param source: :class:`PointData` object containing the source geometry
-    :param receiver: :class:`PointData` object containing the acquisition geometry
-    :param time_order: Time discretization order
-    :param space_order: Space discretization order
+    Construct an gradient operator in an acoustic media.
+
+    Parameters
+    ----------
+    model : Model
+        Object containing the physical parameters.
+    geometry : AcquisitionGeometry
+        Geometry object that contains the source (SpareTimeFunction) and
+        receivers (SparseTimeFunction) and their position.
+    space_order : int, optional
+        Space discretization order.
+    save : int or Buffer, optional
+        Option to store the entire (unrolled) wavefield.
+    kernel : OT2 or 0T4
```

Comment:
these are the values, not the type. Type should be `str, optional`

Reply:
fixed